tests/general/KeywordBearTest.py

```diff
@@ -14,3 +22,99 @@
     invalid_files=("test line todo",
                    "test line warNING"),
     settings={"keywords": "todo, warning"})
+
+
+class KeywordBearDiffTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section("")
+        self.section.append(Setting('language', 'python3'))
+        self.section.append(Setting('keywords', '\# TODO, \#todo'))
+        self.python_uut = KeywordBear(self.section, Queue())
+
+        self.annotation_bear_result_type = namedtuple('result', ['contents'])
+        self.text = ['# todo 123']
+        self.dep_results = {
+            'AnnotationBear': [
+                self.annotation_bear_result_type({'comments': ()})
+            ]
+        }
+        self.dep_results = {'AnnotationBear': HiddenResult(
+            'AnnotationBear', {'comments': ()})}
+
+    def test_todo(self):
+        text = ['a == b']
+
+        with execute_bear(self.python_uut, "F", text,
```

Comment:
`F` is the filename, right? could you just place `"filename"` or `"Filename"` in there, makes it better understandable on first sight than `F` :)